addons/web/static/src/core/popover/popover_container.js

```diff
@@ -42,7 +43,8 @@ class PopoverController extends Component {
         }
     }
     onClickAway(ev) {
-        if (this.target.contains(ev.target) || ev.target.closest(".o_popover")) {
+        if (this.target.contains(ev.target)
+            || ev.target.closest(`.o_popover[popover-id="${this.props.id}"]`)) {
```

Comment:
Feels strange to need an ID for that.

Not an owl expert yet but doesn't this controller have some kind of reference to its root DOM element?

Reply:
I don't think so
they already use the kind of stuff to close the popover (in the popover service)

Reply:
I see a lot of occurrences on "this.root.el", can you double-check? Would be a bit cleaner.

Reply:
Houla :D

Reply:
Well.. fine then :D